robocopkaka/bootcamp-29-project
# #157732026: Edit events via modals

#### What does this PR do?
edit events via modals
#### Description of Task to be completed?
* add an edit modal to the `EventsListWithImage` component
* add a delete case for events to the `singleCenterReducer`
* pass extra props to the EventsListWithImage component inside the `SingleCenter` component
#### How should this be manually tested?
* clone this repo
* cd into the project folder
* run npm install to install all dependecies
* run `npm run start:all` to start up the application
* go to `localhost:8080` on your browser
* click on `Signup`
* enter your details
* click on `Signup`
* request for admin access
* click on `Centers` on the navbar
* click on the red add button at the bottom right corner
* enter the details of the center
* click on `Add Center`
* add at least ten more centers
* click on `Centers`
* select a center
* click on the red add button at the bottom right corner
* enter the details of the event
* click on Add Event
* you should see the newly created event on the center's page
* click on the edit icon on any event
* update the details of the event
* click on `Update Event`
* the details of the event should be updated
#### Any background context you want to provide?
a page refresh has to be done to reflect the update. need to clean up how my reducers are structured
#### What are the relevant pivotal tracker stories?(if applicable)
#157732026
